.gitignore

```diff
@@ -2,3 +2,4 @@ blink-core/
 blink-idl/
 parsed-idl/
 /node_modules
+.idea
```

Comment:
nah.. this line is to protect us from Misko not to protect Misko from our commits.

I'm fine with keeping it in.